src/index.js

```diff
@@ -122,16 +133,18 @@ module.exports = {
     'no-script-url': 'error',
     'no-self-compare': 'error',
     'no-sequences': 'error',
-    'no-shadow-restricted-names': 'error',
     'no-shadow': 'off',
-    'no-spaced-func': 'off',
+    'no-shadow-restricted-names': 'error',
+    'no-spaced-func': 'error',
     'no-sync': 'error',
-    'no-this-before-super': 'error',
     'no-throw-literal': 'error',
     'no-trailing-spaces': 'error',
     'no-undef-init': 'error',
     'no-undefined': 'off',
-    'no-underscore-dangle': 'off',
+    'no-underscore-dangle': ['error', {
+      allow: ['_'],
+      allowAfterThis: true
```

Comment:
This was more like a personal choice. I'm assuming that underscores can be used as prefix for private variables and those should only be accessed directly in their scope (and have some kind of accessor for the outside), thus allowing something like `this._foo = 'bar';`.